crates/nu-command/src/network/post.rs

```diff
@@ -344,6 +382,53 @@ fn helper(
     }
 }
 
+pub fn value_to_json_value(v: &Value) -> Result<nu_json::Value, ShellError> {
```

Comment:
i'm kind of wondering if we should just reference this same function in `to json` instead of duplicating the code or relocate that function somewhere else where it's more easily accessed?

Reply:
it's in two nested private modules right now, I don't know where it would make sense to put it